plugins/jobs/src/js/components/form/GeneralFormSection.tsx

```diff
@@ -1,5 +1,5 @@
 import { Trans } from "@lingui/macro";
-import React, { Component } from "react";
+import * as React from "react";
 import { Tooltip } from "reactjs-components";
 
 import Icon from "#SRC/js/components/Icon";
```

Comment:
When you update with master, you can replace all the `Icon` components used here with `InfoTooltipIcon` ("#SRC/js/components/form/InfoTooltipIcon")